src/components/Supports/index.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+// import { StyledComponentProps } from 'styled-components';
+// import { ThemeInterface } from '../../helpers/style/theme';
+import { StyledWrapper, Title, MediaWrapper } from './index.styles';
+import patreonImg from '../../assets/images/patreon.svg';
+import merchImg from '../../assets/images/merch-store.png';
+import amazonImg from '../../assets/images/amazon-store.png';
+import Button from '../Button';
+
+export default class Supports extends Component {
+  render() {
+    return (
+      <>
+        <Title>SUPPORT THE CODING TRAIN</Title>
```

Comment:
Prefer "Support the Coding Train" - our current font just looks upper case but they're technically different

Reply:
And you never know if we ever change the font sometime down the road.